core/src/main/java/org/mule/runtime/core/privileged/exception/EventProcessingException.java

```diff
@@ -43,6 +44,13 @@ public CoreEvent getEvent() {
     return event;
   }
 
+  /**
+   * @return the Mule component that causes the failure if known, {@code null} otherwise.
+   */
+  public Component getFailingComponent() {
+    return null;
```

Comment:
Something missing here?

Reply:
no, this is overridden by MessagingException that does have this info. I needed this method accessible through a Privileged API to use it in `ExpectedError`